packages/react-hooks/src/useDebouncedValue.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+
+export const DEFAULT_DEBOUNCE_MS = 250;
+
+/**
+ * Debounces a value. Returns the value after the debounce time has elapsed.
+ * @param value Value to debounce
+ * @param debounceMs Amount of time to debounce
+ * @returns The debounced value
+ */
+export function useDebouncedValue<T>(
+  value: T,
+  debounceMs = DEFAULT_DEBOUNCE_MS
```

Comment:
Why provide a default here if we're not using it?

I think it might make more sense for the hook to not include a default since it's more of a util. Just like lodash debounce has no default

Reply:
Yeah fair, I'll pull out the default.